constants.py

```diff
@@ -545,6 +545,40 @@
 
 kMotorBaseKey = "motors"
 
+# waypoint setter constraints
+kMaxWaypointTranslationalVelocity = kMaxForwardLinearVelocity
+kMaxWaypointTranslationalAcceleration = kMaxWaypointTranslationalVelocity * 3
+
+kPossibleWaypoints = [
```

Comment:
What unit is this? I'm a bit worried that the rounding would make it a bit difference if it's in meters

Reply:
PossibleWaypoints? its in [meters, meters, radians]